cmake/modules/FindOATH.cmake

```diff
@@ -8,8 +8,9 @@ find_path(OATH_INCLUDE_DIR
   oath.h
   PATHS
   /usr/include
+  /usr/include/liboath
```

Comment:
why would you want to add this line? we include `oath.h` by `#include <liboath/oath.h>`, so i think we should not include `liboath` in the `PATHS`?

Reply:
@tchaikov 
I added it because otherwise my jenkins build fail in CMake.
see: http://cephdev.digiware.nl:8180/jenkins/job/ceph-master/2196/console
Exactly as per what is required in the find path.
I only added '/usr/local/include/liboath'
and rearanged the includes to have "regular" and "local" grouped.

Other solution in to test:
```
find_path(OATH_INCLUDE_DIR
   liboath/oath.h
   PATHS
   /usr/include
   /usr/local/include
```

Reply:
@wjwithagen thanks for the explanation. i think the right way is the "other solution". i.e. 
```cmake
find_path(OATH_INCLUDE_DIR
   liboath/oath.h
   PATHS
   /usr/include
   /usr/local/include
```

Reply:
@tchaikov 

Right, I'll fix up the PR.